2018/spring/Sterne-MW@11am/roiman-n.md

```diff
@@ -0,0 +1,31 @@
+#markdown is scary
+##really, been coding for a while, never understood it
```

Comment:
Make sure to check Markdown formatting. For instance there needs to be a space between # and the heading. I would check out [this cheat sheet.](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)